app/controllers/dashboards_controller.rb

```diff
@@ -14,7 +14,7 @@ def search
     # n+1 join here
     if params[:search].present?
       @results = Patient.search params[:search],
-                                [current_line.try(:to_sym) || lines]
+                                lines: [current_line.try(:to_sym) || lines]
```

Comment:
I *think* this might lead to a behavior change, but it seems more correct

Reply:
It could! but I did a check through the codebase and Eli caught all the instances 🎉

Reply:
slash it took me a couple of Big Red Failing Tests to figure out lol